pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -60,6 +61,11 @@
                                     <%= question.title %></a>
                                 <%- include('../partials/issueBadge', {count: question.open_issue_count, issueQid: question.qid}); %>
                             </td>
+                            <td class="align-middle">
+                                <% if (question.thumbnail) { %>
+                                <image src="<%= urlPrefix %>/course_admin/question_thumbnail/<%= question.qid %>" onerror="this.style.display='none'" width="64" height="64"/>
```

Comment:
is the `onerror` needed now that we are checking the thumbnail's existence ahead of time?